_posts/2023-05-26-correlated-subqueries-in-sql.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: post
+title:  "(Correlated) Subqueries in SQL"
```

Comment:
Maybe something along the lines of:
"(Correlated) Subqueries in SQL or: How I changed one line in my query and now it has been running for 48 hours"

Reply:
Kind of like "Dr. Strangelove or: How I Learned to Stop Worrying and Love the Bomb"